packages/menus/src/HelpDrawerWidget/components/HelpDrawerWidget.tsx

```diff
@@ -1,5 +1,6 @@
 import Typography from '@material-ui/core/Typography'
 import { observer } from 'mobx-react'
+import { getRoot, IAnyStateTreeNode } from 'mobx-state-tree'
```

Comment:
We need to avoid `getRoot` in anything that's a plugin, since the root might vary from product to product. If a plugin needs to access it, it will need to be on the session (although a session may just end up pointing to that attribute on the root, depending on how the product's model is organized).